TestModels/RecursiveShape/Makefile

```diff
@@ -0,0 +1,47 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+CORES=2
+
+ENABLE_EXTERN_PROCESSING=1
+RUST_BENERATED=1
+
+include ../SharedMakefile.mk
+
+PROJECT_SERVICES := \
+	SimpleRecursiveShape
+
+TRANSLATION_RECORD_PYTHON := \
```

Comment:
Does this pass in Python??

Reply:
No. neither does a lot of other test models. We should do this as part of cleanup or some other PR.  I added it in sim.